scripts/Test-TasksInLocalObsidian.ps1

```diff
@@ -0,0 +1,53 @@
+﻿[CmdletBinding()]
```

Comment:
I am using it to help me work on this PR. So there will be a chicken egg scenario. Ill make a separate PR and if it goes in first then this will disappear from this PR.

Reply:
See #683